### ISSUE TITLE
Weird colors ProtonMail material theme

### ISSUE THREAD

**Issue by Attoy** [Snoobyx/protonmail-material-themes#1]
Hey there, I installed [this](https://raw.githubusercontent.com/Snoobyx/protonmail-material-themes/master/css/protonmail_material.css) theme about a week ago, this morning my webmail looks weird like [that](https://is.gd/cgqh6p) , is it something ProtonMail side or the theme is no longer working? Just FYI I reset the theme and installed it again just to be sure, same weird behaviour.

**Comment by endetti** [comment]
The same problem applies to me. It seems that the style is no longer compatible due to changes introduced in the new version (3.12.23) of ProtonMail.

**Comment by Attoy** [comment]
@endetti I didn't even notice there's a new version! I hope @Snoobyx will release a new fixed version of the theme/css.

**Comment by Snoobyx** [comment]
@endetti @Attoy yeah with the new version ProtonMail changed a lot of things in the front-end, but I will rewrite the theme soon. :)

**Comment by Attoy** [comment]
@snoobyx Awesome, thanks!
